src/snek/terminal.rs

```diff
@@ -27,8 +27,9 @@ pub(crate) struct Terminal {
 
 impl Terminal {
   pub fn new() -> Self {
-    let raw_stdout = std::io::stdout().into_raw_mode().unwrap();
-    let stdout = HideCursor::from(AlternateScreen::from(raw_stdout));
+    let stdout = std::io::stdout().into_raw_mode().unwrap();
+    let stdout = AlternateScreen::from(stdout);
+    let stdout = HideCursor::from(stdout);
```

Comment:
Was handy when I was testing stuff and needed to disable the `AlternateScreen` - this will probably come back into focus when I look into error logging etc